tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckDates.java

```diff
@@ -58,6 +58,7 @@
 
 public class CheckDates extends FactoryCheckCLDR {
     private static final boolean DEBUG = false;
+    private static final boolean DISABLE_DATE_ORDER = false;
```

Comment:
@macchiati do you want it true (disabled), or false (not disabled)?

Reply:
Damn, I'd switched back and forth to compare times. Will reset.

Reply:
Good catch, thanks!

Reply:
```java
    private static final boolean DISABLE_DATE_ORDER = CldrUtility.getProperty("DISABLE_DATE_ORDER", true);
```